### PR TITLE
LPS-152169 Move js code from display_settings.jsp

### PR DESCRIPTION
**Pull request by DiegoHu97**

### CODE REVIEW

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/DisplaySettings.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	const displayStyleSelect = document.getElementById(
+		`${namespace}displayStyle`
+	);
+
+	function showHiddenFields() {
+		const displayStyle = displayStyleSelect.value;
+
+		const hiddenFields = document.querySelectorAll('.hidden-field');
+
+		Array.prototype.forEach.call(hiddenFields, (field) => {
```
Comment (victorg1991) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/js/DisplaySettings.js:25]: We should use `Array.from(hiddenFields).forEach((field`... it's frinedlier to use and doesn't need to use the prototype here :)

**Comment by Tim-Cao** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:forward

**Pull request closed without merging**